rln/src/circuit.rs

```diff
@@ -101,6 +143,9 @@ pub fn vk_from_folder(resources_folder: &str) -> Result<VerifyingKey<Curve>> {
         ))?)?;
         Ok(verifying_key)
     } else if let Some(_zkey) = zkey {
+        #[cfg(feature = "arkzkey")]
+        let (proving_key, _matrices) = arkzkey_from_folder(resources_folder)?;
+        #[cfg(not(feature = "arkzkey"))]
         let (proving_key, _matrices) = zkey_from_folder(resources_folder)?;
```

Comment:
maybe we can abstract this loading into another function where we hide the
`#[cfg(feature = "arkzkey")]`